.github/workflows/tests.yml

```diff
@@ -7,6 +7,12 @@ on:
   pull_request:
     branches:
       - main
+
+# Cancel concurrent runs if they are from the same branch
```

Comment:
This works, in that there is no duplicate run for a branch push that is under PR. But the github UI shows the cancel as a failure. The PR check still succeeds, but it looks like something failed, even though it didn't really. I'm not sure this cure is better than the ailment. Chime in here if you have an opinion.

Reply:
I'd rather have "clean" PR checks...